consensus/hotstuff/committees/consensus_committee.go

```diff
@@ -65,19 +66,20 @@ func NewConsensusCommittee(state protocol.State, me flow.Identifier) (*Consensus
 }
 
 func (c *Consensus) Identities(blockID flow.Identifier, selector flow.IdentityFilter) (flow.IdentityList, error) {
-	return c.state.AtBlockID(blockID).Identities(filter.And(
+	il, err := c.state.AtBlockID(blockID).Identities(filter.And(
 		filter.IsVotingConsensusCommitteeMember,
 		selector,
 	))
+	return il, convertError(err)
```

Comment:
I don't think that we can reach a `protocol.IdentityNotFoundError` error here, as we only drop identities that do not pass the filter; but don't require any particular identity to be present.